src/dev/build/lib/config.ts

```diff
@@ -104,12 +120,26 @@ export class Config {
   }
 
   /**
-   * Return the list of Platforms we are targeting, if --this-platform flag is
-   * specified only the platform for this OS will be returned
+   * Return true if a supported Platform is specified
+   */
+  hasSpecifiedPlatform() {
+    return this.darwinX64 || this.linuxArm64 || this.linuxX64;
+  }
+
+  /**
+   * Return the list of supported Platforms, if --all-platform flag is specified.
+   * Return the targeted Platform, if --target-platform flag is specified.
+   * If no flag, only the platform for this OS will be returned.
    */
```

Comment:
Do we require one of the flags to be provided?

Reply:
Yeah I think we should just be explicit about what the behavior is.

Reply:
Will modify the comment and will update the DEVELOPER_GUIDE.